app/_locales/en/messages.json

```
@@ -1841,6 +1841,10 @@
     "message": "Your $1 has been added to your account.",
     "description": "This message is shown after a swap is successful and communicates the exact amount of tokens the user has received for a swap. The $1 is a decimal number of tokens followed by the token symbol."
   },
+  "swapTokenBalanceUnavailable": {
+    "message": "The balance of $1 is unavailable",
```

I'm not sure about this wording :thinking:. When I first read this, I interpreted it as "You have insufficient balance", i.e. the balance that you need for this transaction is not available because you don't have enough of this token.

Elsewhere in this case we've shown the message "We had trouble loading your token balances.". Something along those lines might be clearer here, like
```suggestion
    "message": "We were unable to retrieve your $1 balance",
```